.github/workflows/pr_lint_and_test.yml

```diff
@@ -1,9 +1,7 @@
 name: on-push-event
```

Comment:
Can we also update this to be the same as the file name?

Reply:
right will do